Sources/Publish/API/AnyDateFormatter.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  AnyDateFormatter.swift
+//  
+//
+//  Created by Dustin Pfannenstiel on 3/10/20.
+//  Copyright (c) Dustin Pfannenstiel, 2020
+//  MIT license, see LICENSE file for details
+//
+
+import Foundation
+
+/// A common interface to abstract common methods and properties present
+/// in all date formatter types.
+public protocol AnyDateFormatter: Formatter {
+    /// The time zone for the receiver.
+    var timeZone: TimeZone! { get set }
+    /// The date format string used by the receiver.
+    var dateFormat: String! { get }
+
+    /// Returns a date representation of a given string interpreted using the receiver’s current settings.
+    /// - parameter string: The string to parse.
+    func date(from string: String) -> Date?
+    /// Returns a string representation of a given date formatted using the receiver’s current settings.
+    /// - parameter date: The date to format.
+    func string(from date: Date) -> String
+}
+
+/// Apply `AnyDateFormatter` to `DateFormatter`
```

Comment:
This comment is unnecessary, since the code is completely self-explanatory.

Reply:
Digging into the PR notes I found a (better) version of this protocol exists as part of your [Codextended library](https://github.com/JohnSundell/Codextended/blob/8d7c46dfc9c55240870cf5561d6cefa41e3d7105/Sources/Codextended/Codextended.swift#L149). The major difference is that this version conforms to `Formatter`.  I'm pretty comfortable dropping that requirement.

I still think that `AnyDateFormatter` has a valuable place in *Publish* to support ISO 8601.  Including this protocol is redundant.  Not including it requires an `@import Codextended` anyplace we use it, regardless of other comments. I'd like your thoughts.